frontend/app/actions/ActionRefiners.scala

```diff
@@ -33,7 +33,7 @@ object ActionRefiners {
 
   type SubReqWithPaid[A] = SubscriptionRequest[A] with PaidSubscriber
   type SubReqWithFree[A] = SubscriptionRequest[A] with FreeSubscriber
-  type SubReqWithSub[A] = SubscriptionRequest[A] with Subscriber
+  type SubReqWithSub[A] = SubscriptionRequest[A] with actions.Subscriber
```

Comment:
scala 2.13 has changed the resolution rules from 2.12.
We have the same class name for two classes.  One is in the same package as this, one is in the controllers package which has a wildcard import.
Scala 2.12 preferred the local one, wheras scala 2.13 prefers the imported one.

In reality we should avoid this situation, but I preferred this solution to aliasing or renaming the class.